packages/twenty-front/src/modules/object-metadata/hooks/useGetObjectMetadataItemById.ts

```diff
@@ -0,0 +1,28 @@
+import { useRecoilValue } from 'recoil';
+
+import { CustomError } from '@/error-handler/CustomError';
+import { objectMetadataItemsState } from '@/object-metadata/states/objectMetadataItemsState';
+import { isDefined } from 'twenty-shared/utils';
+
+export const useGetObjectMetadataItemById = () => {
```

Comment:
I doubt we doubt don't already have similar utils/hooks in the codebase, this seems like something very common

Reply:
Yes, we have `useObjectMetadataItemById`, but this one returns a function.

We have the same design for fields with `useFieldMetadataItemById` and `useGetFieldMetadataItemById`